### PR TITLE
Feature/all dialogues

### PR DESCRIPTION
**Pull request by BenjaminFHV**
The Dialogues & the BackgroundVideo for the SkillTree/Menu is finished and working.

### CODE REVIEW

File: Assets/_Scenes/Development/DialogueScene/DialogueTrigger.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        }
+
+    }
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if (!dialogueIsDone)
+        {
+            //start and activate the dialogue
+            Debug.Log("DialogueShouldStart");
```
Comment (bitingbits) [inline Assets/_Scenes/Development/DialogueScene/DialogueTrigger.cs]: remove debug

File: Assets/_Scenes/Development/DialogueScene/DialogueTrigger.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            dialogueIsDone = true;
+
+			//Freeze the game?! or freeze just player movement? other wise the dialogue frezzes to
+			// Time.timeScale = 0.0f;
```
Comment (bitingbits) [inline Assets/_Scenes/Development/DialogueScene/DialogueTrigger.cs]: remove commented

File: Assets/_Scenes/Development/DialogueScene/DialogueTrigger.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+			//Activate the DialogueCanvas..
+            FindObjectOfType<DialogueRunner>().dialogueCanvas.SetActive(true);
+
+			// Kick off the dialogue at this node.
+            FindObjectOfType<DialogueRunner>().StartDialogue(yarnFileName);
+        }
+        else
+        {
+            //DIalogue already played, do nothing..
+            Debug.Log("DialogueIsDone");
```
Comment (bitingbits) [inline Assets/_Scenes/Development/DialogueScene/DialogueTrigger.cs]: remove debug

File: Assets/_Scenes/Development/TestVideo/PlayVideo.cs [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        //Disable Play on Awake for both Video and Audio
+        videoPlayer.playOnAwake = false;
+        audioSource.playOnAwake = false;
+        audioSource.Pause();
+
+        //We want to play from video clip not from url
+        videoPlayer.source = VideoSource.VideoClip;
+
+        // // Vide clip from Url
+        // videoPlayer.source = VideoSource.Url;
```
Comment (bitingbits) [inline Assets/_Scenes/Development/TestVideo/PlayVideo.cs]: remove commented code

File: Assets/_Scenes/Development/TestVideo/PlayVideo.cs [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+            //Play Sound
+            audioSource.Play();
+            while (videoPlayer.isPlaying)
+            {
+                Debug.Log("Video Time: " + Mathf.FloorToInt((float)videoPlayer.time));
+                yield return null;
+            }
+        }
+
+        Debug.Log("Done Playing Video");
```
Comment (bitingbits) [inline Assets/_Scenes/Development/TestVideo/PlayVideo.cs]: remove debug lines

File: Assets/Scripts/YarnSpinner/Code/YarnFiles/Phase7CallToAdventure.yarn.txt [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    Devon: Yeah, whatever.
+===
+
+title: Option2
+tags: 
+colorID: 0
+position: 682,367
+---
+<<setsprite FaceImage devon>>
+    Devon: Maybe I don't want answers?
```
Comment (Caprico1) [inline Assets/Scripts/YarnSpinner/Code/YarnFiles/Phase7CallToAdventure.yarn.txt]: should be `Devon: Maybe I don't want answers? I’ve tried looking for answers. I’ve done everything I can. And nothing will bring her back.`

Comment (Caprico1) [reply Assets/Scripts/YarnSpinner/Code/YarnFiles/Phase7CallToAdventure.yarn.txt]: After this as well there is a section of dialogue depending on the players decision. That section should display.

```
After acceptance:

Malphas (in Devon’s thoughts): By the way, my name is Malphas. A pleasure to make my introductions. 

Devon (thought): Great it even has a name.

After Refusal 

Malphas: If that is what you want. But remember, my offer still stands when you change your mind eventually.
```

File: Assets/Scripts/Character/MalphasScript.cs [hunk 7]
```diff
@@ -51,10 +49,8 @@ protected override void CharactertInitialize()
         //for development purposes
         skills.Add(new Teleport(skillSettings.teleportSettings, this));
         skills.Add(new Barrier(skillSettings.barierSettings, stats, this));
-        skills.Add(new PhychicScream(skillSettings.phychicScreamSettings, this));
-        skills.Add(new DivineAura(skillSettings.divineAuraSettings, this));
-        skills.Add(new DarkClaw(skillSettings.darkClawSettings, this));
         skills.Add(new DemonicWave(skillSettings.demonicWaveSettings, this));
+        skills.Add(new PhychicScream(skillSettings.phychicScreamSettings, this));
```
Comment (Caprico1) [inline Assets/Scripts/Character/MalphasScript.cs]: Psychic?

File: Assets/Scripts/Character/Abilities/DarkClaw.cs [hunk 8]
```diff
@@ -1,35 +0,0 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public class DarkClaw : ISkill
```
Comment (Caprico1) [inline Assets/Scripts/Character/Abilities/DarkClaw.cs]: Confirm this needs to be deleted!

File: Assets/Scripts/Character/Abilities/DivineAura.cs [hunk 9]
```diff
@@ -1,35 +0,0 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public class DivineAura : ISkill
```
Comment (Caprico1) [inline Assets/Scripts/Character/Abilities/DivineAura.cs]: Confirm this needs to be deleted!

File: Assets/_Scenes/Development/DialogueScene/DialogueTrigger.cs [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+			//Activate the DialogueCanvas..
+            FindObjectOfType<DialogueRunner>().dialogueCanvas.SetActive(true);
+
+			// Kick off the dialogue at this node.
+            FindObjectOfType<DialogueRunner>().StartDialogue(yarnFileName);
+        }
+        else
+        {
+            //DIalogue already played, do nothing..
```
Comment (bitingbits) [inline Assets/_Scenes/Development/DialogueScene/DialogueTrigger.cs]: remove empty else

File: Assets/Scripts/Character/MalphasScript.cs (hunk 7 above)
Comment (bitingbits) [reply Assets/Scripts/Character/MalphasScript.cs]: Physic*

**Pull request merged**